sentences/es/fan_HassTurnOn.yaml

```
@@ -3,7 +3,7 @@ intents:
   HassTurnOn:
     data:
       - sentences:
-          - "enciende [todos] [los] ventiladores <area>"
+          - "<enciende> [todos] [los] ventiladores [(de|en)] <area>"
```

the [(de|en)] is not needed as it's already included in the <area> expansion rule
```suggestion
          - "<enciende> [todos] [los] ventiladores <area>"
```

---

Fixed